schemas/common/external-source-system-audit.schema.json

```diff
@@ -0,0 +1,69 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/common/external-source-system-audit",
```

Comment:
How is this different the existing common/auditable schema - https://github.com/adobe/xdm/blob/master/schemas/common/auditable.schema.json

Are we saying everything in the common/auditable is "internal" audit fields? If I include both of these, am I going to be confused when I see both a xdm:createdDate and a repo:createdDate? Should these fields be named something like xdm:externalCreatedDate? These are just some questions I have.

Reply:
As mentioned this specific mixin will be used as a reference to capture the auditable properties of customers data sets. We have CRM system and other Marketo needs to capture and present when was a record got created Vs Modified and by whom. common/auditable are the fields used meant to be used internally by the AEP services.